providers/amazon/src/airflow/providers/amazon/aws/executors/ecs/utils.py

```diff
@@ -61,7 +61,7 @@ class EcsQueuedTask:
 
     key: TaskInstanceKey
     command: CommandType
-    queue: str
+    queue: str | None
```

Comment:
Can there be a None queue?

Reply:
I swear I was getting type error before, but I don't seem to now. I removed it.